External camel test failed with “pathspec master did not match” error

Testname: external/camel
jdk_impl: openj9
jdk_verison: 11
platform: xlinux
test_link: https://ci.adoptopenjdk.net/job/Grinder_camel/2/console
```
20:17:34       [exec] Step 20/22 : RUN git checkout $CAMEL_TAG
20:17:34       [exec]  ---> Running in 8b7703be1e8a
20:17:35       [exec] [91merror: pathspec 'master' did not match any file(s) known to git
20:17:35       [exec] The command '/bin/sh -c git checkout $CAMEL_TAG' returned a non-zero code: 1
20:17:35       [exec] [0m
20:17:35  
20:17:35  BUILD FAILED
20:17:35  /home/jenkins/workspace/Grinder_camel/aqa-tests/TKG/scripts/build_test.xml:76: The following error occurred while executing this line:
20:17:35  /home/jenkins/workspace/Grinder_camel/aqa-tests/external/camel/build.xml:25: The following error occurred while executing this line:
20:17:35  /home/jenkins/workspace/Grinder_camel/aqa-tests/external/build.xml:87: exec returned: 1
```

--- a/external/camel/test.properties
+++ b/external/camel/test.properties
@@ -2,7 +2,7 @@
  script="camel-test.sh"
  test_command="mvn clean install"
  test_results="testResults"
- tag_version="master"
+ tag_version="main"
  environment_variable="MODE=\"java\""
  debian_packages="git maven"
  debianslim_packages="${debian_packages}"

--- a/external/quarkus_quickstarts/dockerfile/test.sh
+++ b/external/quarkus_quickstarts/dockerfile/test.sh
@@ -49,7 +49,7 @@ mvn -pl !:hibernate-orm-quickstart,!:hibernate-orm-panache-quickstart,\
 !:security-openid-connect-web-authentication-quickstart,\
 !:security-openid-connect-multi-tenancy-quickstart,!:spring-data-jpa-quickstart,\
 !:vertx-quickstart,!:context-propagation-quickstart,!:getting-started-reactive-rest,\
-!:kafka-quickstart,!:neo4j-quickstart clean install
+!:kafka-quickstart,!:neo4j-quickstart,!:rest-client-quickstart,!:rest-client-multipart-quickstart clean install
 test_exit_code=$?
 echo "Build quarkus_quickstarts completed"
 

